api/src/main/java/au/org/aodn/nrmn/restapi/repository/DiverRepository.java

```diff
@@ -21,16 +22,17 @@
 
 @RepositoryRestResource
 @Tag(name = "divers")
-public interface DiverRepository extends JpaRepository<Diver, Integer>, JpaSpecificationExecutor<Diver>, EntityCriteria<Diver> {
+public interface DiverRepository
+        extends JpaRepository<Diver, Integer>, JpaSpecificationExecutor<Diver>, EntityCriteria<Diver> {
 
     @Override
-    @Query("SELECT d FROM  Diver  d WHERE d.initials = :initials")
-    @QueryHints({@QueryHint(name = HINT_CACHEABLE, value = "true")})
-    List<Diver> findByCriteria(@Param("initials")String initials);
-
+    @Query("SELECT d FROM Diver d WHERE d.initials = :initials")
+    @QueryHints({ @QueryHint(name = HINT_CACHEABLE, value = "true") })
+    List<Diver> findByCriteria(@Param("initials") String initials);
 
     @Override
     @RestResource
+    @Query(nativeQuery = true, value = "SELECT * FROM nrmn.diver_ref d ORDER BY (CASE WHEN initials SIMILAR TO '%[a-zA-Z]' THEN 0 ELSE 1 END), d.initials")
```

Comment:
Nice solution.

Reply:
Need to parameterize schema name - see exampels elsewhere by Alex

Reply:
SiteRepository has an example

Reply:
@jonescc updated PR to use `{h-schema}`

Reply:
Just noticed we're overriding the findAll query - I'm just checking to make sure this query will work with the Pageable  etc.

Reply:
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.at-query.native

Reply:
Although I don't think we're actually using the paging at the moment

Reply:
Looks to work OK with examples etc so perhaps just add the count query .